x/valset-pref/types/codec.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+)
+
+func RegisterCodec(cdc *codec.LegacyAmino) {
```

Comment:
`s/validator-set-preference/valset-pref`. Also, I believe the convention is to use the message name as the suffix. e.g. `osmosis/valset-pref/MsgSetValidatorSetPreference`